src/pages/patientView/PatientViewPage.tsx

```diff
@@ -1364,6 +1317,7 @@ export default class PatientViewPage extends React.Component<
                                                     }
                                                     onFilterGenes={
                                                         this
+                                                            .patientViewPageStore
```

Comment:
are you sure this handler belongs on store?

Reply:
Actions on structural variant table are in store. I thought it better to have similar filters at one place

Reply:
reverted back this code